tests/integration/test_vulnerability_detector/test_cpe_helper/data/configuration_template/configuration_cpe_indexing_missing_fields.yaml

```diff
@@ -0,0 +1,75 @@
+- sections:
```

Comment:
Why use three files with the same content ?
- `configuration_cpe_indexing_missing_fields`
- `configuration_cpe_indexing_wrong_tags`
- `configuration_cpe_indexing_wrong_values`

Reply:
Done [here](https://github.com/wazuh/wazuh-qa/pull/3731/commits/044d5b1ff02734c0c25731bde55bd5c3831dc6be)